File: CS/13_Conversion/ToPCL.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using Spire.Pdf;

namespace ToPCL
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Specify the path of the input PDF file.
            string input = @"..\..\..\..\..\..\Data\ToPCL.pdf";

            // Create a new PdfDocument object.
            PdfDocument doc = new PdfDocument();

            // Load the PDF document from the specified input file.
            doc.LoadFromFile(input);

            // Specify the desired file name for the resulting PCL file.
            string output = "ToPCL_result.pcl";

            // Save the loaded PDF document as a PCL file using the specified output file name.
            doc.SaveToFile(output, FileFormat.PCL);

            //Launch the PCL file
            PDFDocumentViewer(output);
        }
        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch {}
        }
    }
}

File: VB/13_Conversion/ToPCL.vb
Imports Spire.Pdf

Namespace ToPCL
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Specify the file path of the input PDF file
			Dim input As String = "..\..\..\..\..\..\Data\ToPCL.pdf"

			' Create a new instance of PdfDocument
			Dim doc As New PdfDocument()

			' Load the PDF file from the specified file path
			doc.LoadFromFile(input)

			' Specify the file path for the resulting PCL file
			Dim output As String = "ToPCL_result.pcl"

			' Save the PDF document to a PCL file
			doc.SaveToFile(output, FileFormat.PCL)

			' Close the PDF document
			doc.Close()

			' Launch the PCL file
			PDFDocumentViewer(output)
		End Sub
		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
